universal-login-wallet/src/core/services/WalletStorageService.ts

```diff
@@ -0,0 +1,40 @@
+import {StorageEntry, StorageService} from './StorageService';
+import {WalletService} from '@universal-login/sdk';
+import {ApplicationWallet, ensure} from '@universal-login/commons';
+import {asObject, asString, Sanitizer} from '@restless/sanitizers';
+
+export class WalletStorageService {
+  private storage: StorageEntry<ApplicationWallet>;
+
+  constructor(
+    private walletService: WalletService,
```

Comment:
Shouldn't dependency be another way around? i.e. WalletService depend on StorageService?

Reply:
Yeah, I can add a wallet storage as a parameter to wallet service that defaults to in-memory storage